### PR TITLE
Test boxplot

### LINKED ISSUE #17: Update boxplot tests

### LINKED ISSUE #21: Add tests using testthat for Boxplot

### PR DESCRIPTION
**Pull request by miyer26**

### CODE REVIEW

File: tests/testthat/test-boxplot.R [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+test_that("The check for input type for y is failing",{
+    expect_error(boxplot(df, origin, "miles_per_gallon",
+                         "y, the column name of y, must be unquoted"))
+})
+
+
+test_that("The check for input type for df is failing",{
+    expect_error(boxplot(c(1,2), miles_per_gallon, origin,
+                         "y, the column name of y, must be unquoted"))
```
Comment (iamMoid) [inline tests/testthat/test-boxplot.R]: You are checking for the wrong error here

**Review by iamMoid — APPROVED** [review]
Looks good. Please review once merged to be double sure.

File: tests/testthat/test-boxplot.R (hunk 1 above)
Comment (miyer26) [reply tests/testthat/test-boxplot.R]: Fixed this

**Review by iamMoid — APPROVED** [review]
Looks good. Do perform a quick test to reassure.

**Pull request merged**